tool/release/Bump.kt

```diff
@@ -0,0 +1,48 @@
+package tool.release
+
+import com.google.api.client.http.ByteArrayContent
+import com.google.api.client.http.GenericUrl
+import com.google.api.client.http.HttpHeaders
+import com.google.api.client.http.javanet.NetHttpTransport
+import java.nio.file.Files
+import java.nio.file.Paths
+
+
+fun postJson(url: String?, authorization: String, accept: String, content: String) {
+    NetHttpTransport()
+    .createRequestFactory()
+    .buildPostRequest(GenericUrl(url), ByteArrayContent.fromString("application/json", content))
+    .setHeaders(HttpHeaders().setAuthorization(authorization).setAccept(accept))
+    .execute()
+}
+
+fun main() {
+    val workspaceDirectory = System.getenv("BUILD_WORKSPACE_DIRECTORY")
```

Comment:
these are good early assertions with informative error messages. I especially liked "Not running from within Bazel workspace". good stuff.